src/main/resources/format-pages.txt

```diff
@@ -2476,6 +2490,20 @@ presenceRating = Fair
 utilityRating = Good
 reader = VeecoReader
 
+[Ventana BIF]
+extensions = .bif
+owner = `Roche Digital Diagnostics <https://diagnostics.roche.com/global/en/home.html/>`_
+bsd = no
+weHave = * some Ventana datasets
+pixelsRating = Good
+metadataRating = Good
+opennessRating = Fair
+presenceRating = Fair
+utilityRating = Good
+privateSpecification = true
```

Comment:
Ratings all look fine, but it might be worth removing ```privateSpecification = true``` since we don't have a specification document for this one.